src/celmodel/mesh.cpp

```diff
@@ -8,19 +8,19 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
+#include <array>
+#include <cstring>
+
+
 #include "mesh.h"
-#include <cassert>
-#include <iostream>
-#include <algorithm>
-#include <Eigen/Core>
-#include <Eigen/Geometry>
 
-using namespace cmod;
-using namespace Eigen;
-using namespace std;
 
+namespace cmod
+{
+namespace
+{
 
-static size_t VertexAttributeFormatSizes[Mesh::FormatMax] =
+constexpr std::size_t VertexAttributeFormatSizes[static_cast<std::size_t>(VertexAttributeFormat::FormatMax)] =
```

Comment:
if c++14 allows `constexpr` functions with `switch` inside then it's better to provide a function instead of array with ugly static_casts

Reply:
And we already had this as a function anyway. Removed the duplication.